addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -149,9 +149,12 @@ export class SuggestionService {
                 .map((member) => member.persona)
                 .filter((persona) => persona.type === "partner");
         } else {
-            partners = Object.values(this.store.Persona.records).filter(
-                (persona) => persona.type === "partner"
-            );
+            partners = Object.values(this.store.Persona.records).filter((persona) => {
```

Comment:
```suggestion
            partners = Object.values(this.store.Persona.records).filter((persona) => {
                return thread.model !== "discuss.channel" && persona.eq(this.store.odoobot) ? false : persona.type === "partner";
            });
```
not sure if is more readable. I think yours might be good already

Reply:
Yep. I think the first one is more readable :)